### DISCUSSION TITLE
What is the purpose of cythonizing wave apps?

### DISCUSSION THREAD

**Discussion by Mathanraj-Sharma** [h2oai/wave#1363] [Q&A]
Normally we cythonize the python packages to increase the performance & to secure the source code.

But what is the purpose of cythonizing wave app? (Even we cythonize the app’s package, end of the day we use `.wave`  artifact instead of `.whl`  for deployment)

cc: @rasika @lasith011 @VijithaEkanayake

**Comment by lo5** [comment]
For obfuscating the source code of proprietary apps (to a certain extent).
